xbmc/input/ButtonTranslator.cpp

```diff
@@ -1195,6 +1195,38 @@ CAction CButtonTranslator::GetGlobalAction(const CKey &key)
   return GetAction(-1, key, true);
 }
 
+bool CButtonTranslator::HasLonpressMapping(int window, const CKey &key)
+{
+  map<int, buttonMap>::const_iterator it = m_translatorMap.find(window);
+  if (it == m_translatorMap.end())
+  {
+    if (window > -1)
+      return HasLonpressMapping(GetFallbackWindow(window), key);
```

Comment:
after a second look I adjusted the fix because we need to do the fallback handling at the top if we don't find a mapping for the given window. Let's move over the discussion to the PR.